.github/workflows/ms.network.virtualnetworks.yml

```diff
@@ -72,94 +67,19 @@ jobs:
     outputs:
       workflowInput: ${{ steps.get-workflow-param.outputs.workflowInput }}
       moduleTestFilePaths: ${{ steps.get-module-test-file-paths.outputs.moduleTestFilePaths }}
+      modulePath: '${{ env.modulePath }}'
```

Comment:
Why's that? Can we not reference the env value directly?

Reply:
Unfortunately, not. Interestingly, env variables can not be passed to reusable workflows

Reply:
How odd. So this, really, is just a workaround to gain access to the module path env variable?

Reply:
Yes, others just add a job at the beginning to just print out env var as job outputs, but since we already have the initialize pipeline job in our case I think it makes more sense to leverage it also for that